### DISCUSSION TITLE
AttributeError: module 'distutils' has no attribute 'version'

### DISCUSSION THREAD

**Discussion by MislavSag** [BenWiseman/sentiment.ai#7] [Q&A]
I have created new conda environent called sentimentai. Then I have installed following packages in the environment:
```
list(numpy = "1.19.5", sentencepiece = "0.1.95", tensorflow = "2.4.1",
    tensorflow_hub = "0.12.0", `tensorflow-text` = "2.4.3")
```
The, when I execute:
```
init_sentiment.ai(envname = "sentimentai")
```
I get an error:
```
Preparing Model
2022-03-14 07:46:33.374721: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
Error in py_run_file_impl(file, local, convert) : 
  AttributeError: module 'distutils' has no attribute 'version'
```

**Comment by BenWiseman** [comment]
Are you running within Rmarkdown or an Rstudio project?

**Comment by MislavSag** [comment]
Rstudio project

**Comment by swnydick** [comment]
Can you post the code that you ran that caused the issue? Ideally this would be a "reproducible example", but anything that is causing the issue would be good. Also, what is your OS and how is conda or virtualenv or whatever you are using installed on your computer? I gave it a shot, but I'm having trouble reproducing the error on a new OS. Either the python/reticulate versions do not align (and the installation crashes) or everything seems to work.
